src/app/shared/pipes/userDisplay.ts

```diff
@@ -0,0 +1,17 @@
+import { Pipe, PipeTransform } from '@angular/core';
+
+interface UserInfos {
+  login: string,
+  firstName :string|null,
+  lastName :string|null,
+}
+
+@Pipe({ name: 'userDisplay' })
+export class UserDisplayPipe implements PipeTransform {
+  transform<T extends UserInfos>(userInfos: T) : string {
+    if (userInfos.firstName === null || userInfos.lastName === null) {
+      return userInfos.login;
+    }
+    return `${userInfos.firstName} ${userInfos.lastName} (${userInfos.login})`;
```

Comment:
The case where only one of firstName/lastName is null is ignored. Probably we should be in the second scenario for that case, but not displaying "null" for the null one.

Reply:
fixed in PR #197